lib/roo_on_rails/sidekiq/settings.rb

```diff
@@ -1,16 +1,19 @@
 module RooOnRails
   module Sidekiq
     class Settings
+      DEFAULT_QUEUES = %w(
+        monitoring
+        realtime
+        within1minute
+        within5minutes
+        within30minutes
+        within1hour
+        within1day
+      ).freeze
+
       def self.queues
-        %w(
-          monitoring
-          realtime
-          within1minute
-          within5minutes
-          within30minutes
-          within1hour
-          within1day
-        ).freeze
+        env_key = 'SIDEKIQ_QUEUES'
+        ENV.key?(env_key) ? ENV[env_key].split(',') : DEFAULT_QUEUES
```

Comment:
I thought of that but doing it in one place would mean doing it in all places i.e. if we assume that a user can add a space after a comma, they might add a space anywhere. Still, I've appended a `strip` to wherever the user can add a space. And I've mentioned the sorting in the README now. - done